lemur/certificates/schemas.py

```diff
@@ -353,6 +355,20 @@ def handle_subject_details(self, data):
                 data.pop("organization", None)
                 data.pop("organizational_unit", None)
 
+    @post_dump
+    def handle_certificate(self, cert):
+        if cert['root_authority']:
```

Comment:
what is the goal of `cert['root_authority']`, how is it different than cert['authority']

Reply:
I need authority for:
- get the plugin name (to run wrap_certificate inside plugin)
- get the authority private_key to sign OpenSSH certificate

Assume that we have:
- one OpenSSH authority with certificate ID 1
- one OpenSSH certificate sign by this authority with certificate ID 2

If I get https://xxxxxx/lemur/api/1/certificates/1:
- 'authority' is None
- 'root_authority is itself

If I get https://xxxxxx/lemur/api/1/certificates/2:
- 'authority' is the current authority
- 'root_authority is None

Reply:
I think I understand where the different point of views are coming from:

for a CA certificate, `root_authority_id` is set to the respective issuer in the Authority table. 
This is how for instance the Authority UI can render the CA certificate:

https://github.com/Netflix/lemur/blob/cb2f3402772212f2b1de980b72a8765ee5c78d45/lemur/authorities/models.py#L51-L56

Reply:
sorry if I might be missing something obvious with the model, or how the schema works, but how come these two are different?

```
    authority = fields.Nested(AuthorityNestedOutputSchema)
    root_authority = fields.Nested(AuthorityNestedOutputSchema)
```